web/src/components/DandiAppBar.vue

```diff
@@ -105,6 +162,22 @@ export default {
       return 'NA';
     },
   },
-  methods: mapActions(['logout', 'selectSearchResult']),
+  methods: {
+    async register_dandiset() {
+      const { name, description } = this;
+      const { status, data } = await this.girderRest.post(`dandi`,
+          stringify({ name: name, description: description}));
```

Comment:
No need to use another library here

```suggestion
      const { status, data } = await this.girderRest.post('dandi', null, { params: { name, description } });
```